src/main/java/com/minecolonies/coremod/entity/ai/citizen/cook/EntityAIWorkCook.java

```diff
@@ -0,0 +1,542 @@
+package com.minecolonies.coremod.entity.ai.citizen.cook;
+
+import com.google.common.reflect.TypeToken;
+import com.minecolonies.api.colony.requestsystem.requestable.Burnable;
+import com.minecolonies.api.colony.requestsystem.requestable.Food;
+import com.minecolonies.api.util.InventoryUtils;
+import com.minecolonies.api.util.ItemStackUtils;
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.coremod.colony.StructureName;
+import com.minecolonies.coremod.colony.Structures;
+import com.minecolonies.coremod.colony.buildings.BuildingCook;
+import com.minecolonies.coremod.colony.buildings.BuildingWareHouse;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.colony.buildings.BuildingWareHouse'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)